src/universal/types/App.types.ts

```diff
@@ -92,9 +92,9 @@ export interface GenericDocument {
 
 export type AltDocumentContent = string | ReactNode;
 
-export interface StatusLineItem {
+export interface StatusLineItem<T = string> {
```

Comment:
Ik denk wel dat we willen forceren dat status in ieder geval een string is toch?
```suggestion
export interface StatusLineItem<T extends string = string> {
```

Reply:
nog een goede thanks